src/content/en/tools/chrome-devtools/_project.yaml

```
@@ -0,0 +1,37 @@
+name: Chrome DevTools
+description: Chrome DevTools are a set of web authoring and debugging tools built into Google Chrome.
+home_url: /web/tools/chrome-devtools/
+buganizer_id: 180451
+color: google-blue
+content_license: cc-apache
+footer_path: /web/_footer.yaml
+google_analytics_ids:
+  - UA-52746336-1
```

```suggestion
- UA-52746336-1
```

TIL: The DevSite linter says this shouldn't be indented.